vocabularies/earth-science-data-category.ttl

```diff
@@ -880,6 +882,7 @@ esdc:lodgement-form a skos:OrderedCollection ;
         esdc:engineering,
         esdc:surveying,
         esdc:drilling-engineering,
+	esdc:resources-engineering,
```

Comment:
Pls consider if the 'resources' should be used in single form?

Reply:
@talebib label as per source material https://vocabs.ardc.edu.au/repository/api/lda/anzsrc-for/resource?uri=http://purl.org/au-research/vocabulary/anzsrc-for/2008/0914

But can certainly change our label if preferred. I think there is room to add some of the narrower terms here as well